### PR TITLE
Fix Messaging, indicate incomprehensible word

### PR DESCRIPTION
**Pull request by kwadrat**
I am unable to correct "activatallow_authing" because
I don't understand this word.

### CODE REVIEW

File: salt/states/ipmi.py [hunk 1]
```diff
@@ -176,13 +176,13 @@ def user_present(name, uid, password, channel=14, callback=False,
...
     ipmi_msg
-        User IPMI Messaginge
+        User IPMI Messaging
         True/False
         user name and password information will be used for IPMI
         Messaging. In this case, 'IPMI Messaging' refers to the ability to
         execute generic IPMI commands that are not associated with a
         particular payload type. For example, if IPMI Messaging is disabled for
-        a user, but that user is enabled for activatallow_authing the SOL
+        a user, but that user is enabled for ??? the SOL
```
Comment (thatch45) [inline salt/states/ipmi.py]: I think it is safe to say that the word in question is "activating"

**Comment by kwadrat** [comment]
What should I do to improve the changes?

**Comment by jfindlay** [comment]
@kwadrat, the remaining lint failures are unrelated and have already been fixed.

**Comment by kwadrat** [comment]
Thank you!

**Pull request merged**